PSScriptAnalyzerSettings.psd1

```diff
@@ -5,7 +5,7 @@
     # Only diagnostic records of the specified severity will be generated.
     # Uncomment the following line if you only want Errors and Warnings but
     # not Information diagnostic records.
-    Severity = @('Error','Warning')
+    Severity = @('Error')
```

Comment:
Could you elaborate a bit why we are decreasing the log level to `Error` only? I think it has a real value to have `Warning` as well. WDYT?